public/app/plugins/datasource/loki/language_provider.ts

```diff
@@ -332,7 +332,9 @@ export default class LokiLanguageProvider extends LanguageProvider {
   }
 
   importFromAbstractQuery(labelBasedQuery: AbstractQuery): LokiQuery {
-    return toPromLikeQuery(labelBasedQuery);
+    const q = toPromLikeQuery(labelBasedQuery);
```

Comment:
We are using here `toPromLikeQuery` from prometheus data source which has `range: boolean` parameter and we set it to `range: true` https://github.com/grafana/grafana/blob/main/public/app/plugins/datasource/prometheus/language_utils.ts#L293 . As this is not true anymore for Loki, we should probably create Loki's own `toLokiQuery` function that doesn't use range/instant param anymore.